features/admin_creates_auction.feature

```diff
@@ -2,6 +2,7 @@ Feature: Admin creates auctions in the admins panel
   As an admin
   I should be able to create auctions
 
+  @background_jobs_off
```

Comment:
when we create an auction is schedules a job for far into the future--a future where the auction has ended and where there are winning and losing vendors. if we keep background jobs on, it gets invoked immediately in the feature spec, and looks for a winning/losing vendor that doesn't yet exist (since the feature only is concerned with auction creation, not ending)